demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -0,0 +1,221 @@
+                const float confidence = data[i * OBJECT_SIZE + 2];
+                const float rc_left    = data[i * OBJECT_SIZE + 3];
+                const float rc_top     = data[i * OBJECT_SIZE + 4];
+                const float rc_right   = data[i * OBJECT_SIZE + 5];
+                const float rc_bottom  = data[i * OBJECT_SIZE + 6];
+                if (image_id < 0.f) { 
+                    break;
+                }
+                if (confidence < 0.25f) { 
+                    continue;
+                }
+
+                cv::Rect rc;
+                rc.x      = static_cast<int>(rc_left   * upscale.width);
+                rc.y      = static_cast<int>(rc_top    * upscale.height);
+                rc.width  = static_cast<int>(rc_right  * upscale.width)  - rc.x;
+                rc.height = static_cast<int>(rc_bottom * upscale.height) - rc.y;
+                out_faces.push_back(rc & surface);
+            }
+        }
+    };
+    GAPI_OCV_KERNEL(OCVPersonDetActionRec, PersonDetActionRec) {
+        static void run(const cv::Mat &in_ssd_local,
+                        const cv::Mat &in_ssd_conf,
+                        const cv::Mat &in_ssd_priorbox,
+                        const cv::Mat &in_ssd_anchor1,
+                        const cv::Mat &in_ssd_anchor2,
+                        const cv::Mat &in_ssd_anchor3,
+                        const cv::Mat &in_ssd_anchor4,
+                        const cv::Mat &in_frame, std::vector<Detections> &out_detections) {
+            const float *local_data    = reinterpret_cast<float*>(in_ssd_local.data);
+            const float *det_conf_data = reinterpret_cast<float*>(in_ssd_conf.data);
+            const float *prior_data    = reinterpret_cast<float*>(in_ssd_priorbox.data);
+
+            std::vector<float*> action_conf_data;
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor1.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor2.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor3.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor4.data));
+                
+            const cv::Size frame_size = in_frame.size();
+            std::unique_ptr<ActionDetector> detector(new ActionDetector);
+            detector->GetPostProcResult(local_data, det_conf_data, prior_data, action_conf_data, frame_size, out_detections);
+        }
+    };
+}// namespace custom
+    
+int main(int argc, char* argv[]) {
+    cv::CommandLineParser cmd(argc, argv, keys);
+    if (cmd.has("help")) {
+        cmd.printMessage();
+        return 0;
+    }
+
+    cv::GComputation pp([]() {
+        cv::GMat in;
+        cv::GMat detections = cv::gapi::infer<custom::Faces>(in);
+        cv::GArray<cv::Rect> faces = custom::PostProc::on(detections, in);
+
+        cv::GMat location;
+        cv::GMat detect_confidences;
+        cv::GMat priorboxes;
+        cv::GMat action_con1;
```

Comment:
Here. I wanted to use `cv::GArray anchors = {action_con1, action_con2, action_con3, action_con4};`. And put GArray in post processing instead four GMat.
But it didn't work (failed experiment =) ).